src/frontend/src/environments/environment.prod.ts

```diff
@@ -3,5 +3,6 @@ export const environment = {
   baseApiUrl: '/api/v1/',
   loginApiUrl: '/api/login/',
   usersApiUrl: '/api/v1/users/',
-  eventsApiUrl: '/api/v1/events/'
+  eventsApiUrl: '/api/v1/events/',
+  refreshApiUrl: '/api/refresh'
```

Comment:
I'm not sure how to force refresh the token but I think this will fail without a trailing slash. Django is **real** picky about that. :(